qemu/tests/qemu_guest_agent.py

```diff
@@ -416,21 +416,24 @@ def gagent_check_set_user_password(self, test, params, env):
         old_password = params.get("password", "")
         new_password = params.get("new_password", "123456")
         crypted = params.get("crypted", "") == "yes"
+        error_context.context("Change guest's password.")
         try:
             if crypted:
                 self.gagent.set_user_password(new_password, crypted)
```

Comment:
`username` should be passed to here as well.

Reply:
username is a default parameter in function defination, does it need to reference ?
def set_user_password(self, password, crypted=False, username="root")

Reply:
If you introduce the param to the cfg file, it means you let the users use their own value. So, in order to make it work properly, you have to pass it to the function.